File: SoloLearn Codes/No Numerals/NoNumerals.cs
using System.Linq;
using System;

namespace SoloLearn
{
    class Program
    {
        static void Main(string[] args)
        {
            string[] arr = { "zero", "one", "two", "three", "four", "five", "six", "seven", "eight", "nine", "ten" };
            Console.WriteLine(string
                .Join(" ", Console.ReadLine()
                .Split()
                .Select(w => char.IsDigit(w[0]) ? arr[Convert.ToInt32(w)] : w)
                .ToArray()));
            ;
        }
    }
}

File: SoloLearn Codes/No Numerals/no_numerals.swift
let arr = ["zero", "one", "two", "three", "four", "five", "six", "seven", "eight", "nine", "ten"];
print(readLine()!.split(separator: " ")
    .map{ t in t.prefix(while: { "0"..."9" ~= $0 }) == "" ? String(t) : arr[Int(t)!] }
    .joined(separator: " "))